src/Components/Form/Form.js

```diff
@@ -36,17 +38,9 @@ function Form({contacts}) {
 
   return (
     <form className='trip-form'>
-      <input
-        type='text'
-        name='input'
-        className='start-point'
-        placeholder='Start Point'
-      />
-      <input
-        type='text'
-        name='input'
-        className='end-point'
-        placeholder='End Point'
+      <SearchLocationInput 
```

Comment:
Nice work here. We'll have to research how to work around the API issue and what our options are.